modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
 --
 =====
 
-== Cloud architecture
+== Redpanda Cloud architecture
 
-Redpanda Cloud uses a control plane and data plane architecture. The control
-plane configures and maintains cluster agents, which are
-self-contained and independent of the control plane.
+When you first sign up for a Redpanda account, Redpanda creates an organization for you. Your organization contains all your Redpanda resources, including your clusters and networks. Within your organization, Redpanda creates a default namespace as a place to group your resources. You can rename this namespace, and you can create more namespaces. For example, you may want one namespace for a certain team, or you may want different namespaces for production and testing. 
+
+For high availability, Redpanda Cloud uses a control plane and data plane architecture. The control plane enforces rules in the data plane, and the data plane is where your Redpanda cluster resides. The term _data plane_ is used interchangeably with _cluster_. 
+
+The following diagram shows a BYOC architecture, where you deploy the Redpanda data plane in your own VPC. Redpanda manages it on the control plane with an agent. 
+
+image::shared:d_c_plane.png[Data plane and control plane]
+
+In the user interface breadcrumbs, when you're at the organization (org) level or the namespace level, but you haven't yet selected a cluster, you're in the control plane. This is where you can select, create, and delete clusters, namespaces, and networks. When you're at the cluster level working with topics, consumer groups, and connectors, you're in the data plane. 
```

Comment:
This reads a little awkwardly to me. The reference to breadcrumbs doesn't feel necessary. You might rephrase this a little for clarity.